Examples/Selection Monitor/chemdraw-addin-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "description": "This is an example of ChemDraw add-in that demostrates the usage of the selection API.",
```

Comment:
type: `demostrates`  ==> `demonstrates`

Reply:
I think it should be `This is an example ChemDraw add-in that demonstrates the usage of the selection API.`